crypto/bbs.go

```diff
@@ -0,0 +1,33 @@
+package crypto
+
+import (
+	"crypto/rand"
+	"crypto/sha256"
+
+	bbs "github.com/hyperledger/aries-framework-go/pkg/crypto/primitive/bbs12381g2pub"
+)
+
+// GenerateBBSKeyPair https://w3c-ccg.github.io/ldp-bbs2020
+func GenerateBBSKeyPair() (*bbs.PublicKey, *bbs.PrivateKey, error) {
```

Comment:
Did you mean to have this be part of this PR?

Reply:
nope